mgmt/LocalManager.cc

```diff
@@ -900,6 +900,10 @@ LocalManager::startProxy()
 
     real_proxy_options.append(proxy_options, strlen(proxy_options));
 
+    if (!strstr(proxy_options, "-M")) { // Make sure we're starting the proxy in mgmt mode
```

Comment:
This should really be a `#define` string to guarantee consistency and then use `sizeof(M_OPTION_STRING)-1` in the `append` call.